### ISSUE TITLE
Wrong network settings for cortex plugin impact the whole instance

### ISSUE THREAD

**Issue by torelab** [MISP/MISP#3198]
### Work environment

| Type of issue             | Bug
| OS version (server)    | Ubuntu Server 16.04 

| MISP version / git hash   | v2.4.90/v2.4.89
| Browser                           | Opera/Firefox/Chrome
Cortex Version: 2.0.4

### Actual behavior
Serious performances impact on MISP, the GUI slows down when trying to retrieve attributes by events and when trying to act on MISP settings the interface often get painfully stuck in "Loading" (even by just loading the panel).

### Steps to reproduce the behavior
Set wrong network settings to Cortex plugin.

**Comment by syloktools** [comment]
I confirm that this is happening to me as well.

**Comment by SteveClement** [comment]
This is indeed a bug.
@yalbertorosso @robertnixon2003 I suppose by malformed you mean something really wrong, I tried with "%"

I did try with a correct URI without a CORTEX running, and it worked ok.

**Comment by syloktools** [comment]
It no longer effects the whole instance. I think they fixed that a while back. It would try to connect for a long time before timing out.
		
		

		Get Outlook for iOS
	




On Thu, Jul 19, 2018 at 3:33 PM -0400, "Steve Clement" <notifications@github.com> wrote:












This is indeed a bug.

@yalbertorosso @robertnixon2003 I suppose by malformed you mean something really wrong, I tried with "%"


I did try with a correct URI without a CORTEX running, and it worked ok.
